docs/source/use_cases/analyze_reads/calculate_coverage.rst

```diff
@@ -45,6 +48,13 @@ write access, you may use it.  If not, you can do the following to create one:
 The ``id`` that is generated in the response output is the output dataset id you should use when running
 the pipeline.
 
+Download ALPN
+-------------
+
+Running this pipeline locally requires that you have the ALPN jar that matches your JRE version
+on your computer.  See the `ALPN documentation <http://www.eclipse.org/jetty/documentation/9.2.10.v20150310/alpn-chapter.html>`_ for a table of which ALPN jar to use.
+
+You will not need to provide the ALPN flag or jar if you run the pipeline on Google cloud instead of locally.
```

Comment:
Unfortunately no, having that flag and having it registered as true (hence the log call in main) is the only way for GCE to know to use ALPN, which is required for gRPC.  I'll forward you the doc that Sam sent me with the instructions, although it looks like you may have already seen it.